stories/data/DataMocks.js

```diff
@@ -637,6 +637,297 @@ export const pathListChunk = {
     ],
 };
 
+export const listingSuccessResp = {
```

Comment:
I found myself adding to the mocks for the data listing quite a bit, so I moved it into the DataMock.js file to clean up the data listing Story a bit. Hopefully that's okay.